service/src/main/java/com/derpgroup/derpwizard/resource/AlexaResource.java

```diff
@@ -65,75 +67,28 @@ public AlexaResource(MainConfig config, Environment env) {
    * @return The message, never null
    */
   @POST
-  public SpeechletResponseEnvelope doAlexaRequest(SpeechletRequestEnvelope request){
-    if(request == null || request.getRequest() == null){
+  public SpeechletResponseEnvelope doAlexaRequest(@NotNull @Valid SpeechletRequestEnvelope request){
```

Comment:
SpeechletRequestEnvelope is a third party class that we can't control, so the `@Valid` annotation has nothing to work with.

Also, `@NotNull` doesn't do anything to a parameter representing the body of the request - it's already not allowed to be null, and the same error is thrown with or without it.